presidentielles/sondages.json

```diff
@@ -0,0 +1,2094 @@
+            "barnier": "Michel Barnier candidat LR/UDI"
+        },
+        "reports_2017": {
+            "pecresse":
+                {
+                    "votes_2017": ["Jean-Luc Mélenchon", "Benoit Hamon", "Emmanuel Macron", "François Fillon", "Marine Le Pen"],
+                    "intentions_2022": ["Jean-Luc Mélenchon", "Yannick Jadot", "Anne Hidalgo", "Emmanuel Macron", "Valérie Pécresse", "Marine Le Pen", "Éric Zemmour"],
+                    "valeurs": [
+                        [38, 22, 16, 4, 0, 7, 2],
+                        [6, 28, 45, 14, 1, 2, 1],
+                        [2, 8, 8, 68, 7, 3, 2],
+                        [2, 2, 1, 22, 54, 7, 10],
+                        [0, 2, 2, 1, 3, 78, 13]
+                    ]
+                },
+            "bertrand":
+                {
+                    "votes_2017": ["Jean-Luc Mélenchon", "Benoit Hamon", "Emmanuel Macron", "François Fillon", "Marine Le Pen"],
+                    "intentions_2022": ["Jean-Luc Mélenchon", "Yannick Jadot", "Anne Hidalgo", "Emmanuel Macron", "Xavier Bertrand", "Marine Le Pen", "Éric Zemmour"],
+                    "valeurs": [
+                        [33, 23, 16, 5, 2, 6, 2],
+                        [6, 27, 44, 10, 9, 1, 0],
+                        [1, 8, 9, 68, 9, 1, 3],
+                        [0, 2, 1, 22, 55, 7, 12],
+                        [0, 1, 2, 2, 5, 77, 12]
+                    ]
+                },
+            "wauquiez":
+                {
+                    "votes_2017": ["Jean-Luc Mélenchon", "Benoit Hamon", "Emmanuel Macron", "François Fillon", "Marine Le Pen"],
+                    "intentions_2022": ["Jean-Luc Mélenchon", "Yannick Jadot", "Anne Hidalgo", "Emmanuel Macron", "Laurent Wauquiez", "Marine Le Pen", "Éric Zemmour"],
+                    "valeurs": [
+                        [35, 22, 15, 4, 1, 8, 2],
+                        [6, 31, 43, 12, 2, 3, 0],
+                        [1, 9, 9, 72, 3, 3, 2],
+                        [0, 2, 0, 30, 46, 9, 11],
+                        [0, 1, 3, 1, 2, 78, 14]
+                    ]
+                },
+            "barnier":
+                {
+                    "votes_2017": ["Jean-Luc Mélenchon", "Benoit Hamon", "Emmanuel Macron", "François Fillon", "Marine Le Pen"],
+                    "intentions_2022": ["Jean-Luc Mélenchon", "Yannick Jadot", "Anne Hidalgo", "Emmanuel Macron", "Michel Barnier", "Marine Le Pen", "Éric Zemmour"],
+                    "valeurs": [
+                        [34, 22, 15, 5, 1, 8, 2],
+                        [6, 29, 47, 12, 1, 2, 0],
+                        [2, 8, 9, 69, 6, 2, 2],
+                        [2, 4, 0, 27, 44, 9, 11],
+                        [0, 1, 3, 2, 1, 80, 14]
+                    ]
+                }
+        },
+        "premier_tour": {
+            "pecresse": {
+                "intentions_exprimees": null,
+                "note": "11% certains d'aller voter n'expriment pas d'opinion",
```

Comment:
That means we can compute `intentions_exprimees`: `(100% - 11%) * echantillon`

Reply:
mmmm you don't know what % of population is sure to vote necessarily so I think we'll need to be very explicit with our names